mozilla_version/maven.py

```diff
@@ -15,11 +14,11 @@ class MavenVersion(BaseVersion):
 
     is_snapshot = attr.ib(type=bool, default=False)
 
-    _VALID_ENOUGH_VERSION_PATTERN = re.compile(r"""
+    _VALID_ENOUGH_VERSION_PATTERN = r"""
         ^(?P<major_number>\d+)
```

Comment:
Nit: Hm, in here we kept the `^` and `$` while in the other clases we removed them. Should we remove them from here too, for consistency?

Reply:
Later edit: reading https://github.com/mozilla-releng/mozilla-version/pull/66/files#diff-a14382edb70f4cbdd6a25d8b5dcea7c5R42, isn't the MavenVersion going to have duplicate `^..$` signs? since `is_within_bigger_string` will likely be `False`?

Reply:
Oops! Thanks for spotting it. I removed the duplicate `^..$` signs